src/Orc.Controls/Controls/StepBar/Models/StepBarItemBase.cs

```diff
@@ -0,0 +1,41 @@
+﻿namespace Orc.Controls.Controls.StepBar.Models
+{
+    using System;
+    using Catel;
+    using Catel.Data;
+    using Catel.MVVM;
+
+    public abstract class StepBarItemBase : ModelBase, IStepBarItem
+    {
+        private IViewModel _viewModel;
+
+        public IViewModel ViewModel
```

Comment:
Remove, this model should be much more simplified. It *just* represents a step inside a step bar.